web/client/reducers/geostory.js

```diff
@@ -125,6 +127,15 @@ const getIndexToInsert = (array, position) => {
  * }
  *
  */
+let INITIAL_STATE = {
```

Comment:
moving the initial state here you associate the jsdoc of the reducer to the INITIAL_STATE... but the comment is related to the reducer.  Please move this in the original position.

Reply:
i just wanted to have the initial state near to the reducer to avoid scrolling up..

Reply:
I understand the purpose... Consider the documentation as a part of the function.